handlers/dataset_test.go

```diff
@@ -89,7 +92,15 @@ func TestDatasetHandlers(t *testing.T) {
 
 				Convey("When the dataset page is rendered", func() {
 					w, req := generateRecorderAndRequest()
-					DatasetPage(mockZebedeeClient, mockRend, mockFilesAPIClient)(w, req)
+
+					cfg, err := config.Get()
```

Comment:
**COMMENT 1**
You don't need to do this as you already have got `cfg` in this test at line 53 which contains `SupportedLanguages`. You can remove lines 96 to 98